frontend/test/metabase/scenarios/custom-column/reproductions/21135-cc-same-name-as-existing-column.cy.spec.js

```diff
@@ -0,0 +1,56 @@
+import { restore } from "__support__/e2e/cypress";
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
+
+const { PRODUCTS_ID, PRODUCTS } = SAMPLE_DATABASE;
+
+const questionDetails = {
+  name: "21135",
+  query: {
+    "source-table": PRODUCTS_ID,
+    limit: 5,
+    expressions: { Price: ["+", ["field", PRODUCTS.PRICE, null], 2] },
+  },
+};
+
+describe.skip("issue 21135", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.createQuestion(questionDetails, { visitQuestion: true });
+
+    switchToNotebookView();
+  });
+
+  it("should handle cc with the same name as the table column (metabase#21135)", () => {
+    cy.findAllByTestId("notebook-cell-item")
+      .contains("Price")
+      .click();
+    cy.button("Update").click();
+
+    previewCustomColumnNotebookStep();
+
+    // We should probably use data-testid or some better selector but it is crucial
+    // to narrow the results to the preview area to avoid false positive result.
+    cy.get("[class*=PreviewRoot]").within(() => {
+      cy.findByText("Rustic Paper Wallet");
+    });
```

Comment:
It would be great if we added assertion for real Price and the CC Price (issue 21131), so just an extra validation of results.

Reply:
Done.
Thank you for the feedback.